knowledge-base/google/clusterfuzzlite/actions/run_fuzzers/action-security.yml

```diff
@@ -0,0 +1,9 @@
+name: 'run-fuzzers'
+github-token:
+  action-input:
+    input: github-token #CHECKOUT: https://github.com/google/clusterfuzzlite/blob/main/actions/run_fuzzers/action.yml#L28
+  permissions:
+    actions: read
+    actions-reason: to fetch the artifacts #CHECKOUT: https://github.com/google/oss-fuzz/blob/0bf87a10c4c1df2514dd82023e4da2d2d26b2e1c/infra/cifuzz/filestore/github_actions/github_api.py#L47
```

Comment:
This also needs `contents: read` to clone the repo.

Reply:
@varunsh-coder I added the required `permission`